src/mstart.sh

```diff
@@ -46,7 +46,7 @@ if [ $? -ne 0 ]; then
 fi
 
 pos=`echo $pos | cut -d: -f1`
-base_port=$((6800+pos*20))
+base_port=$((6900+pos*20))
```

Comment:
why do we need this?

Reply:
With 3 cluster port start to clash if we don't change. I don't recall what clash with what exactly though but I can check if you want

Reply:
As the mstart.sh is used not only by rbd mirror tests, it would be good to have this change as a separate commit with an explanation why we need this.

Reply:
Tested it again and with 3 cluster I have an osd that start on 6860 and a mon from the third cluster too.